packaging_docker/testnet/azure/secret-node-azure-template/createUiDefinition.json

```diff
@@ -215,7 +215,7 @@
             "defaultValue": "115aa0a629f5d70dd1d464bc7e42799e00f4edae@51.132.225.152:26656",
             "constraints": {
               "required": true,
-              "regex": "^([a-f0-9]{40}@([0-9]{1-3}\\.){3}[0-9]{1-3}:[0-9]{1,5},\\w?){1,10}$",
+              "regex": "^([a-f0-9]{40}@([0-9]{1,3}.){1,3}[0-9]{1,3}:[0-9]{2,5},?){1,10}$",
```

Comment:
don't you still need to say `\\.` instead of just `.` there?

Reply:
nah, it doesn't need to be escaped